docs/tutorials/validate_data.rst

```diff
@@ -0,0 +1,447 @@
+The DataContext is configured via a yml file stored in a directory called great_expectations;
+the configuration file as well as managed expectation suites should be stored in version control.
+
+Instantiating a DataContext loads our project configuration and all its resources.
+
+::
+
+    context = ge.data_context.DataContext()
+
+To read more about DataContext, see: :ref:`data_context`
+
+
+
+2. List Data Assets
+-------------------
+
+A Data Asset is data you can describe with expectations.
+
+.. content-tabs::
+
+    .. tab-container:: tab0
+        :title: pandas
+
+        A Pandas datasource generates data assets from Pandas DataFrames or CSV files. In this example the pipeline processes NPI data that it reads from CSV files in ``npidata`` directory into Pandas DataFrames. This is the data you want to describe and specify with expectations. That directory and its files are a data asset, named "NPI data" (based on the directory name).
+
+    .. tab-container:: tab1
+        :title: pyspark
+
+        A Spark datasource generates data assets from Spark DataFrames or CSV files. In this example the pipeline processes NPI data that it reads from CSV files in ``npidata`` directory into Pandas DataFrames. This is the data you want to describe and specify with expectations. If the example read the data into Spark DataFrames, we would think of this data asset as "data from the npidata directory that we read into Spark DataFrames" and give it a name "NPI data".
+
+    .. tab-container:: tab2
+        :title: SQLAlchemy
+
+        A SQLAlchemy datasource generates data assets from tables, views and query results.
+
+        * If the data resided in a table (or view) in a database, it would be accessible as a data asset with the name of that table (or view).
+        * If the data did not reside in one table ``npidata`` and, instead, the example pipeline ran an SQL query that fetched the data (probably from multiple tables), the result set of that query would be accessible as a data asset. The name of this data asset would be up to us (e.g., "npidata" or "npidata_query").
+
+
+Great Expectations' ``jupyter_ux`` module has a convenience method that lists all data assets and expectation suites known to a Data Context:
+
+.. code-block:: python
+
+    great_expectations.jupyter_ux.list_available_data_asset_names(context)
+
+Here is the output of this method when executed in our example project:
+
+.. image:: ../images/list_data_assets.png
+    :width: 600px
+
+``npidata`` is the short name of the data asset. Full names of data assets in a DataContext consist of three parts, for example: ``data__dir/default/npidata``. You don't need to know (yet) how the namespace is managed and the exact meaning of each part. The :ref:`data_context` article describes this in detail.
+
+
+3. Pick a data asset and and expectation suite
```

Comment:
Copy the same down from above. (note typo `and and` here)

Reply:
+1